eclass/qt5-build.eclass

```diff
@@ -676,9 +676,10 @@ qt5_base_configure() {
 
 	popd >/dev/null || die
 
-	if [[ ${QT5_MINOR_VERSION} -ge 8 ]]; then
+	if [[ ${QT5_MINOR_VERSION} -ge 8 && ${QT5_MINOR_VERSION} -lt 9 ]]; then
```

Comment:
this is always false, we don't have any 5.8 ebuilds, so you may as well delete the whole if.

Reply:
Ack, done.